.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'Message to comment on stale issues. If none provided, will not mark issues stale'
```

Comment:
On further reflection, we should probably update this and the `stale-pr-message` with something more AM specific.